### PR TITLE
Modify rule S4487: Adapt to LaYC

### PR DESCRIPTION
**Pull request by frederic-tingaud-sonarsource**
* Cfamily directory was removed as the rule is not implemented and there is no plans to implement it

### CODE REVIEW

File: rules/S4487/compliant.adoc [hunk 1]
```diff
@@ -1,6 +1,4 @@
-=== Compliant solution
-
-[source,text]
+[source,java,diff-id=1,diff-type=compliant]
 ----
```
Comment (vilchik-elena) [inline rules/S4487/compliant.adoc:2]: Probably if you remove unimplemented languages this file will be useless

File: rules/S4487/python/rule.adoc [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
+* "class-private" attributes have a name starting with at least two underscores and ending with at most one underscore. These attributes' names will be automatically mangled to avoid collision with subclasses' attributes. For example ``++__myattribute++`` will be renamed as ``++_classname__myattribute++``, where ``++classname++`` is the attribute's class name without its leading underscore(s). They shouldn't be used outside of the class defining the attribute.
+
+This rule raises an issue when a class-private attribute (two leading underscores, max one underscore at the end) is never read inside the class. It optionally raises an issue on unread attributes prefixed with a single underscore. Both class attributes and instance attributes will raise an issue.
+
+
+== How to fix it
```
Comment (vilchik-elena) [inline rules/S4487/python/rule.adoc:13]: Why did you decide to go for a separate tab for Python? The fix does not look more complex than for .Net. I would go for a single tab

**Review by vilchik-elena — CHANGES_REQUESTED** [review]

File: rules/S4487/python/rule.adoc (hunk 2 above)
Comment (frederic-tingaud-sonarsource) [reply rules/S4487/python/rule.adoc:13]: @guillaume-dequenne-sonarsource The python version of the description is very different from the other languages and contains scary underscores in the code example. Do you want to have second pass on it on your side or are you happy with what I have here?

File: rules/S4487/cfamily/rule.adoc [hunk 3]
```diff
@@ -4,6 +4,8 @@ include::../description.adoc[]
 
 include::../noncompliant.adoc[]
 
+{outro}
+
 include::../compliant.adoc[]
 
 == Resources
```
[the remark this answers is not in the document]
Comment (frederic-tingaud-sonarsource) [reply rules/S4487/cfamily/rule.adoc]: Done

File: rules/S4487/compliant.adoc (hunk 1 above)
Comment (frederic-tingaud-sonarsource) [reply rules/S4487/compliant.adoc:2]: I'll run the script again for inlining, once the removal is done

**Review by vilchik-elena — APPROVED** [review]
If ignoring Python, LGTM

**Pull request merged**